Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -36,6 +37,11 @@ enum class MaskType : u8 {
     HasLasso = 3
 };
 
+enum class ViewportMode : u32 {
+    EHB = 0x80,
+    HAM = 0x800
+};
```

Comment:
So, if you want bitwise operators on an  enum, we have a fancy macro for that:

AK_ENUM_BITWISE_OEPRATORS(ViewportMode);

That adds the bitwise operators, as well as handy dandy overloads for `has_flag()`

`has_flag(enum_object, ViewportMode::EHB)` etc

Reply:
Nice! I updated my PR to use it.